### ISSUE TITLE
Null values in request body are replaced with empty objects, preventing POST requests from being called properly

### ISSUE THREAD

**Issue by yoonslee** [IBM/openapi-to-graphql#263]
Hi @Alan-Cha or anyone,

Sample "POST" mutation with payload:
```
mutation {
  sampleMutation(requestBodyInput: [{name: "Yoyo", price: null}, {name: "Apple", price: null}])
}
```

From logging an HTTP request => `request` package:
```
{
    "json": true,
    "jsonReplacer": [Function: jsonReplacer],
    "method": "post",
    "url": "<URL>",
    "headers": {
      "content-type": "application/json",
      "accept": "application/json",
      "Authorization": "<AUTH TOKEN>"
    },
    "body": "[{ 'name: 'Yoyo', price: {} }, { 'name': 'Apple', price: {} }]"
  }
```

The replacing of the price values from `null` to `{}` is causing my POST request to fail. I attempted to place a jsonReplacer function inside the `requestOptions` to replace the empty objects back to `null` before sending the request, but no luck there. Any quick tips on how I can preserve the `null` values, or get the jsonReplacer function to properly pass? I have no control over this API that I am attempting to hit FYI. Also, I assume the `null` values are being replaced because of something to do with GraphQL?

```
requestOptions: {
  json: true,
  jsonReplacer: (key, value) => {
    if (Object.entries(value).length === 0 && value.constructor === Object) return null
    return value
  },
},
```

**Comment by Alan-Cha** [comment]
Hey! This is a bug. Thanks for bringing it to our attention. I just fixed it with #264.

We'll pull it in tomorrow (pair review).

**Comment by yoonslee** [comment]
⚡ Appreciate the fix!
